src/Artsy/Router/buildServerApp.tsx

```diff
@@ -31,7 +35,7 @@ export function buildServerApp(config: RouterConfig): Promise<Resolve> {
         const historyMiddlewares = [queryMiddleware]
         const resolver = new Resolver(relayEnvironment)
         const render = createRender({})
-        const headTags = []
+        const headTags = [<style type="text/css">{MediaStyle}</style>]
```

Comment:
This makes sense, but I wonder if there might be somewhere less hidden?

Reply:
Not sure where. Also keep in mind that it’s easy to find this place if you go to the symbol in the main setup file and use TS to find references to it.